myhpom/templates/myhpom/content/why-plan.html

```diff
@@ -0,0 +1,237 @@
+{% extends "myhpom/base.html" %}
+{% load scribbler_tags %}
+{% load staticfiles %}
+
+{% block main_container_class %}myhpom-page__container{% endblock main_container_class %}
+
+{% block main_content %}
+    <section class="content-page__hero-banner" style="background-image: url({% static 'myhpom/img/why-plan/father-at-home-with-tablet-video-chatting_web.jpg' %})"></section>
+    <section class="why-plan-overview">
+        <div class="container">
+            <div class="row">
+                <div class="col-12">
+                    <h1 class="why-plan-overview__title">
+                        Why Plan?
+                    </h1>
+                    <h2 class="why-plan-overview__subtitle">
+                        A quick guide to how an advance care plan
+                        keeps your care in check.
+                    </h2>
+                    <div class="mt-3 mb-4 why-plan-overview__text">
+                        {% scribble 'why-plan' 'overview' %}
```

Comment:
As I read the django-scribbler docs, the first argument is the "slug", which is the tag that's unique on this page for this scribble, and the second (optional) argument is the url, which defaults to the url of the page. In this case, I think we can omit the url and just use "overview" as the slug, or swap the arguments, if I'm understanding scribbler correctly. (And so throughout the template)